command/loadbot/generator/erc_generators.go

```diff
@@ -0,0 +1,62 @@
+package generator
+
+import (
+	"encoding/hex"
+	"fmt"
+
+	"github.com/0xPolygon/polygon-edge/crypto"
+)
+
+// call ERC20 contract method and encode parameters
+func NewERC20Generator(params *GeneratorParams) (*ContractTxnsGenerator, error) {
+	gen := &ContractTxnsGenerator{}
+
+	gen.BaseGenerator = BaseGenerator{
+		failedTxns: make([]*FailedTxnInfo, 0),
+		params:     params,
+		signer:     crypto.NewEIP155Signer(params.ChainID),
+	}
+
+	buf, err := hex.DecodeString(params.ContractArtifact.Bytecode)
+	if err != nil {
+		return nil, fmt.Errorf("unable to decode bytecode, %w", err)
+	}
+
+	gen.contractBytecode = buf
+	gen.contractBytecode = append(gen.contractBytecode, params.ConstructorArgs...)
+
+	if gen.encodedParams, err = params.ContractArtifact.ABI.Methods["transfer"].Encode(
+		[]string{params.RecieverAddress.String(),
+			"3",
+		}); err != nil {
+		return nil, fmt.Errorf("cannot encode ERC20 transfer method params: %w", err)
+	}
+
+	return gen, nil
+}
+
+// call ERC721 contract method and encode parameters
+func NewERC721Generator(params *GeneratorParams) (*ContractTxnsGenerator, error) {
+	gen := &ContractTxnsGenerator{}
+
+	gen.BaseGenerator = BaseGenerator{
+		failedTxns: make([]*FailedTxnInfo, 0),
+		params:     params,
+		signer:     crypto.NewEIP155Signer(params.ChainID),
+	}
+
+	buf, err := hex.DecodeString(params.ContractArtifact.Bytecode)
+	if err != nil {
+		return nil, fmt.Errorf("unable to decode bytecode, %w", err)
+	}
+
+	gen.contractBytecode = buf
+	gen.contractBytecode = append(gen.contractBytecode, params.ConstructorArgs...)
+
+	if gen.encodedParams, err = params.ContractArtifact.ABI.Methods["createNFT"].Encode(
+		[]string{"https://realy-valuable-nft-page.io"}); err != nil {
```

Comment:
Typo here `realy` -> `really`

Reply:
Fixed the typo in 0d0dc3410f11c71b098c4238344680a86d0d070e